content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -53,6 +53,14 @@ same node. This means that setting a _safe_ sysctl for one pod
 - must not allow to gain CPU or memory resources outside of the resource limits
   of a pod.
 
+{{< note >}}
+Starting from Kubernetes version 1.23, kubelet supports the use of "/" or "." as separators for sysctl.
```

Comment:
```suggestion
Starting from Kubernetes version 1.23, the kubelet supports the use of either `/` or `.`
as separators for sysctl names.
```

Reply:
Thanks for your suggestion, I will update it.